repository_service_tuf_api/__init__.py

```diff
@@ -196,7 +196,12 @@ def is_bootstrap_done():
     """
     Check if the boot is done.
     """
-    settings_repository.reload()  # reload the settings
+    # Reload the settings
+    # The reload is required because the settings object is created in the
+    # `app.py`'s initialization. The `settings_repository.get_fresh() doesn't
+    # correctly reload it because that and the settings_repository.reload() do
+    # the job.
+    settings_repository.reload()
```

Comment:
Yes, the settings object.

At `app.py`, it is not a problem because it is how the RSTUF application is started. It just started and got the fresh `settings_repository`.
We don't use it at multiple places, only in the `app.py` when starting the application.